src/core_plugins/kibana/public/dashboard/dashboard_consts.js

```diff
@@ -0,0 +1,5 @@
+
+export const DashboardConsts = {
+  ADD_TO_DASH_PARAM: 'addToDashboard',
+  ADD_VIS_PARAM: 'addVisualization'
```

Comment:
I'm having a hard time telling these two apart because the names are so similar.

I think these would be a little clearer if they were `ADD_VISUALIZATION_TO_DASHBOARD_MODE_PARAM` and `NEW_VISUALIZATION_ID_PARAM`. I know this seems verbose, but that's only because our nouns happen to have lots of letters in them -- in terms of using as few words as possible to communicate the maximum amount of information, I think they're pretty efficient!